data/map altera.txt

```diff
@@ -4292,56 +4292,56 @@ system Zephyrius
-	spaceport `The spaceport seems to have been replaced almost completely as well and is much more open than the original Quarg design, as the Alterans have nothing to hide and even the military section is only partially off-limits to civilians. In order to travel around the spaceport, or rather the entire ringworld, a system of local teleporters and magnetically levitated trains has been built.`
-	spaceport `	The experience on this ring is nothing like that of a Quarg ring, it seems busier and much more welcoming, stemming mostly from the more open Alteran design, but the people and talking present. Several shipyards and outfitters can be found around the spaceport and a nice bar has placed itself right next to one of those teleporters.`
+	description `Just from looking at its exterior, it is obvious that this is a finished Quarg ringworld, though it is clearly not in the possession of the Quarg anymore. While it looks just like any other ringworld from outside, on the inside it becomes appearant that the Alterans must have taken over this ring ages ago. Large portions of the Quarg interior and technology have been augmented or replaced with Alteran designs and the gravitational force is quite a bit stronger, even compared to human standards.`
+	spaceport `The spaceport seems to have been replaced almost completely as well and is much more open than the original Quarg design, as the Alterans have nothing to hide and even the military section is only partially off-limits to civilians. In order to travel around the spaceport and the rest of the ringworld, a system of local teleporters and magnetically levitated trains has been built.`
+	spaceport `	The experience on this ring is nothing like that on a Quarg ring, it seems busier and much more welcoming, a result of the open Alteran architecture and the constant presence of people welcoming a chat. Several shipyards and outfitters can be found around the spaceport and a nice bar is situated next to one of the teleporters.`
 	shipyard "Ancient Advanced"
 	outfitter "Ancient Advanced"
 
 planet Celestis
 	landscape land/city7
-	description `The Alteran capital of Celestis, one of the most populated Alteran worlds, the pre war census coming to over 12 billion. Its proportions seem to be fairly similar to those of the Earth, it consists of highly populated futuristic cities full of towering skyscrapers, a few bigger mountain ranges and large oceans. Unlike Earth though, it is not overpopulated and surprisingly, the oceans appear to be in pristine condition.`
-	description `	Even though the Alteran Empire is a monarchy, the seat of government is not immediately visible and appears to be very down to earth on closer inspection. This is reflected in the respect for the monarchy, not caused by fear, but by a mutual understanding, teamwork and the meritocratic ways of the government.`
```

Comment:
With this rewrite I am not happy. I understood what you wanted to say, but had difficulties expressing it myself.